specs/CustomScrollbar.md

```diff
@@ -0,0 +1,93 @@
+Custom Scrollbar Style
+===
+
+# Background
+Developers would like to be able to set the scrollbar to Windows fluent overlay style and 
+match the Windows theme. Some customers have used browser flags but they change names and behavior and are not reliable. Thus we want
+to provide an extendable API that allows developers to set scrollbar style. Currently, 
+only Windows fluent overlay are supported, adding new styles in the future will not introduce
+any breaking changes.
+
+# Examples
+## WinRT and .NET   
+```c#
+/// Create WebView Environment with option
+async void CreateEnvironmentWithOption()
+{
+    CoreWebView2EnvironmentOptions options = new CoreWebView2EnvironmentOptions();
+    options.CustomScrollbarStyle = ScrollbarStyle.WindowOverlay;
```

Comment:
Is there precedence for `Scrollbar` vs `ScrollBar`? Xaml is all `ScrollBar`. I think that's right; dictionary.com doesn't know "scrollbar"

Reply:
In the web world, `scrollbar` appears to come as one word (https://developer.mozilla.org/en-US/search?q=scrollbar). But I don't mind ScrollBar to be consistent with the rest of Windows.

Reply:
Please change to ScrollBar